lib/components/Input/CheckboxInput.tsx

```diff
@@ -123,17 +112,12 @@ export const CheckboxInput = React.memo((props: CheckboxInputProps) => {
                     className={css('checkbox-fill')}
                     attr={props.attr?.indeterminateFill}
                 />
-                <ThemeConsumer>
-                    { theme => 
-                        <Icon
-                            icon='checkMark'
-                            size={IconSize.xsmall}
-                            className={css('checkbox-checkmark')}
-                            attr={props.attr?.checkmarkIcon}
-                            color={theme && theme.colorTextBtnPrimaryRest}
-                        />
-                    }
-                </ThemeConsumer>
+                <Icon
+                    icon='checkMark'
+                    size={IconSize.xsmall}
+                    className={css('checkbox-checkmark')}
+                    attr={props.attr?.checkmarkIcon}
```

Comment:
are we removing the colors?

Reply:
We are removing styled components since we can use css custom properties to do that without the extra dependency, everything else should stay the same